Dockerfile

```diff
@@ -48,6 +48,7 @@ LABEL "com.github.actions.description"="API testing"
 LABEL "com.github.actions.icon"="home"
 LABEL "com.github.actions.color"="red"
 
+LABEL "description"="This is an API testing tool that supports HTTP, gRPC, and GraphQL." 
```

Comment:
The format should be `LABEL org.opencontainers.image.description DESCRIPTION`

Reply:
done!